Objects/EventItem/EventItem.gd

```diff
@@ -0,0 +1,28 @@
+extends Node2D
+
+export(int) var gold_reward = 65
+export(float) var move_speed = 100
+export(Vector2) var start_position = Vector2()
+
+
+
+func _ready():
+	var value = randi()%800+160
+	set_start_position(value,550)
+
+func _process(delta):
+	move(delta)
+	
+func move(delta):
+	position.x -= move_speed * delta
+
+func set_start_position(var x, var y):
```

Comment:
writing 'var' before variable name is unneaded for function parameters. I think, we should write every function in the same way, so I recomend to remove that 'var' words.

Reply:
Right